Source/DafnyLanguageServer.Test/Lookup/ReferencesTest.cs

```diff
@@ -0,0 +1,267 @@
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.Dafny.LanguageServer.IntegrationTest.Extensions;
+using Microsoft.Dafny.LanguageServer.IntegrationTest.Util;
+using OmniSharp.Extensions.LanguageServer.Protocol;
+using OmniSharp.Extensions.LanguageServer.Protocol.Document;
+using OmniSharp.Extensions.LanguageServer.Protocol.Models;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Microsoft.Dafny.LanguageServer.IntegrationTest.Lookup {
+  public class ReferencesTest : ClientBasedLanguageServerTest {
+    private async Task<LocationContainer> RequestReferences(
```

Comment:
Nit: I would move this and the next method to the bottom of the file, so the file is more callgraph topologically ordered.